bin/tarantool-lsp

```diff
@@ -26,9 +26,10 @@ package.setsearchroot(lsp_root)
 _G._ROOT_PATH = lsp_root
 
 -- For debug purposes
--- box.cfg{
---     log = "log.txt"
--- }
+box.cfg{
+ log = "log.txt",
```

Comment:
You don't need to call `box.cfg` because you don't need the database. And it always creates useless snapshots.